woocommerce/woocommerce
# Run all unit tests in HPOS context

### All Submissions:






### Changes proposed in this Pull Request:



Closes #36376 

This PR adds a new workflow for the unit tests suite that makes it run with HPOS enabled. Earlier we were running unit tests for only post data stores. 






### How to test the changes in this Pull Request:



Check that there is a workflow called `Run unit tests on PR / PHP 7.4 WP latest HPOS (pull_request)` running in the unit test list. It will fail currently, but will pass after all dependent PRs are merged



### Other information:




### FOR PR REVIEWER ONLY:
